subcommands/keys/ca_show.go

```diff
@@ -169,6 +169,9 @@ func prettyPrint(cert string) {
 		case *ecdsa.PublicKey:
 			fmt.Println("\t\tNIST CURVE:", pub.Curve.Params().Name)
 			fmt.Print("\t\t\t")
+
+			//FIXME: Fix deprecation and remove the nolint
+			//nolint: staticcheck: SA1019: elliptic.Marshal has been deprecated since Go 1.21: for ECDH, use the crypto/ecdh package. This function returns an encoding equivalent to that of PublicKey.Bytes in crypto/ecdh. (staticcheck)
```

Comment:
This one should not have happened; probably related to an uninteded linter upgrade.

The thing is: Golang moves too fast for us.
So we usually stick to stable releases, which are sometimes 1-2 major versions behind.
For example, the highest Golang version supported by LmP is `1.19`, soon will be `1.20`.
Still, we have some users trying to compile Fioctl inside LmP, and we don't want to upset them by using a linter which prints warnings for an unsupported Golang version.

Reply:
>  and we don't want to upset them by using a linter which prints warnings for an unsupported Golang version.

**None of the changes proposed in this PR affect end users; the linter solely raises warnings for us/maintainers**. Could you please clarify your comment? I apologize, but it seems there might have been a misunderstanding, and I want to ensure we're on the same page.

> Still, we have some users trying to compile Fioctl inside LmP

**Linters are not checked during the build/compile process of the tool,** so these issues wouldn't arise in such scenarios. Could you please clarify this point further?

Reply:
By downgrading the linter version the warning is not raised. 
So, I removed the comment.

Reply:
Anyway, your new changes are good, so this discussion is no longer needed.